KayKnows-frontend/src/app/tree/tree.component.html

```diff
@@ -1,10 +1,14 @@
 <li *ngFor="let dataObject of data">
     <a class="tooltip tooltip-new {{dataObject.type}}" (mouseover)="setHighlighted(dataObject.band_id)" (mouseleave)="setHighlighted(-1)" [ngStyle]="{
         'border' : getColour(dataObject),
-        'background-color' : getHighlighted() === dataObject.band_id ? getBackgroundColour(dataObject) : 'white',
-        'color' :  getHighlighted() === dataObject.band_id ? 'white' : (dataObject.type === 'role' || dataObject.type === 'family') ? 'black' : 'white'
-    }" (click)="dataObject.opened = !dataObject.opened" href="#">
-        {{ dataObject.label }}        <br> <input id=moreInfo (click)="onSelect(dataObject, $event)" type="image" src="https://image.flaticon.com/icons/png/512/51/51941.png">
+        'background-color' : getHighlighted() === dataObject.band_id ? getBackgroundColour(dataObject) : (dataObject.is_important) ? getBackgroundColour(dataObject) : 'white',
```

Comment:
could be
`'background-color' : (getHighlighted() === dataObject.band_id || (dataObject.is_important)) ?  getBackgroundColour(dataObject) : 'white'`

or extract to method